handbook/product-design/product-design.rituals.yml

```diff
@@ -65,5 +65,7 @@
   task: "Pre-sprint prioritization" # 2024-03-06 TODO: Link to responsibility or corresponding "how to" info e.g. https://fleetdm.com/handbook/company/product-groups#making-changes
   startedOn: "2024-02-27" 
   frequency: "Triweekly"
+  description: "Align on priorities for upcoming sprint."
```

Comment:
@noahtalerman, I think you meant to delete lines 70 and 71 right?

Reply:
Yes! Thanks. Done